.circleci/config.yml

```diff
@@ -14,18 +14,14 @@ commands:
       - run: &install_cmake
           name: Install cmake
           no_output_timeout: 5m
-          # OpenEXR requires CMake 3.12. We also download CMake 3.20, which
-          # we'll use later for the JS build only.
+          # OpenEXR requires CMake 3.12.
           command: |
               echo $(date +%F) > ./date
               echo $(git ls-remote https://github.com/facebookresearch/habitat-lab.git HEAD | awk '{ print $1}') > ./hablab_sha
               cat ./hablab_sha
               wget https://cmake.org/files/v3.12/cmake-3.12.4-Linux-x86_64.sh
-              wget https://cmake.org/files/v3.20/cmake-3.20.1-linux-x86_64.sh
               sudo mkdir /opt/cmake312
-              sudo mkdir /opt/cmake320
               sudo sh ./cmake-3.12.4-Linux-x86_64.sh --prefix=/opt/cmake312 --skip-license
-              sudo sh ./cmake-3.20.1-linux-x86_64.sh --prefix=/opt/cmake320 --skip-license
               sudo ln -s /opt/cmake312/bin/cmake /usr/local/bin/cmake
       - run: &install_deps
           name: Install dependencies
```

Comment:
What about "Install Headless Chrome dependencies" below? Maybe that is only for testing web stuff?